src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -191,24 +191,22 @@ public void clickSectionSelectionCheckBox(int instrNum, int sectionLevelIndex, i
         this.sectionSelectionCheckBox(instrNum, sectionLevelIndex - 1, sectionNum - 1).click();
     }
     
-    public void clickViewStudentCheckBoxInSectionLevel(int instrNum, int sectionLevelIndex) {
-        this.sectionLevelPanelCheckBox(instrNum, sectionLevelIndex,
-                     Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_STUDENT_IN_SECTIONS).click();
+    public void clickSectionLevelPrivilegeLink(int instrNum, int sectionLevelIndex, int linkNum) {
+        this.sectionLevelPanelCheckBox(instrNum, sectionLevelIndex, linkNum - 1).click();
     }
     
-    public void clickViewOthersCommentsCheckBoxInSectionLevel(int instrNum, int sectionLevelIndex) {
-        this.sectionLevelPanelCheckBox(instrNum, sectionLevelIndex,
-                     Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_COMMENT_IN_SECTIONS).click();
+    public boolean isSectionLevelPrivilegeLinkClicked(int instrNum, int sectionLevelIndex, int linkNum) {
+        return this.sectionLevelPanelCheckBox(instrNum, sectionLevelIndex, linkNum - 1).isSelected();
     }
     
-    public void clickViewSessionResultsCheckBoxInSectionLevel(int instrNum, int sectionLevelIndex) {
-        this.sectionLevelPanelCheckBox(instrNum, sectionLevelIndex,
-                     Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS).click();
+    public void clickSessionLevelPrivilegeLink(int instrNum, int sectionLevelIndex,
+                                               int sessionIndex, int linkNum) {
+        this.sessionLevelTableCheckbox(instrNum, sectionLevelIndex, sessionIndex, linkNum - 1).click();
     }
     
-    public void clickModifySessionResultCheckBoxInSectionLevel(int instrNum, int sectionLevelIndex) {
-        this.sectionLevelPanelCheckBox(instrNum, sectionLevelIndex,
-                     Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS).click();
+    public boolean isSessionLevelPrivilegeLinkClicked(int instrNum, int sectionLevelIndex,
+                                                      int sessionIndex, int linkNum) {
+        return this.sessionLevelTableCheckbox(instrNum, sectionLevelIndex, sessionIndex, linkNum - 1).isSelected();
```

Comment:
Should be possible, it might make the code cleaner in fact, haven't touched this PR in a while but I believe there was a mix of 0- and 1-based index definitions from the call sites.